apps/web/src/lang/fr.json

```diff
@@ -5499,6 +5235,290 @@
     "defaultMessage": "Langue préférée pour l'examen écrit :",
     "description": "Display text for the preferred written exam language field on users"
   },
+  "/R/S8P": {
```

Comment:
Ya, it's annoying since some of the strings are for select (or multi-select) options which shouldn't be wrapped in abbr tags. However, if you find one missing that's not a select option then flag it :)